wouso/games/teamquest/templates/teamquest/cpanel/list_quests.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'cpanel/index.html' %}
+{% load i18n %}
+{% load django_bootstrap_breadcrumbs %}
+
+{% block sectiontitle %}
+    Team Quests
+    <div class="pull-right">
+        <a class="btn btn-primary" href="#">
+            <span class="glyphicon glyphicon-plus"></span>{% trans 'Add Team Quest' %}
```

Comment:
You should not capitalize team and quest. "Add Team Quest" should be "Add team quest".